src/common/initialize.lua

```diff
@@ -27,6 +28,8 @@ function initialize.initializeANTState(state)
 	local description = encoded.description
 	local keywords = encoded.keywords
 	local owner = encoded.owner
+	local logo = encoded.logo or ""
```

Comment:
Unsure why this needs defensive check and not elsewhere

Reply:
actually this should be the default logo

Reply:
[57aea4e](https://github.com/ar-io/ar-io-ant-process/pull/95/commits/57aea4e8fefb31a8b17b80e3882258b30dcc71ad) - and the defensive check was incorrect, it gets asserted below